### PR TITLE
food overlay now works for food in off hand

### PR DESCRIPTION
**Pull request by LordMZTE**

### CODE REVIEW

**Comment by LordMZTE** [comment]
i think its always been annoying that this mod doesnt show the overlay for food in the players offhand so i added that in (food in the main hand is still prioritized) would you like me to add a config option so this can be turned on and off in case youd prefer that?

File: java/squeek/appleskin/client/HUDOverlayHandler.java [hunk 1]
```diff
@@ -74,7 +74,9 @@ public void onRender(RenderGameOverlayEvent.Post event)
 
 		Minecraft mc = Minecraft.getInstance();
 		PlayerEntity player = mc.player;
-		ItemStack heldItem = player.getHeldItemMainhand();
+		ItemStack heldItem;
+		if(!FoodHelper.isFood(heldItem = player.getHeldItemMainhand()))
```
Comment (squeek502) [inline java/squeek/appleskin/client/HUDOverlayHandler.java]:
```suggestion
		ItemStack heldItem = player.getHeldItemMainhand();
		if(!FoodHelper.isFood(heldItem))
```

Not a fan of assignment in conditionals

**Review by squeek502 — CHANGES_REQUESTED** [review]
Thanks! No config option needed. Just one style nitpick.

**Review by squeek502 — APPROVED** [review]

**Comment by LordMZTE** [comment]
thanks for merging!

**Pull request merged**